packages/fluentui/docs/src/prototypes/chatPane/chatPaneHeader.tsx

```diff
@@ -84,20 +81,21 @@ class ChatPaneHeader extends React.PureComponent<ChatPaneHeaderProps> {
           }))}
           styles={{ marginRight: '20px' }}
         />
-        {['team-create', 'more'].map((name, index) => (
-          <Icon
-            key={`${index}-${name}`}
-            name={name}
-            outline
-            tabIndex={0}
-            styles={{
-              fontWeight: 100,
-              margin: 'auto',
-              ...(!index && { margin: 'auto 1.6rem auto auto' }),
-            }}
-            variables={siteVars => ({ color: siteVars.colors.grey[350] })}
-          />
-        ))}
+        {[TeamCreateIcon, MoreIcon].map((IconComponent, index) => {
+          return (
+            <IconComponent
+              key={`${index}-${name}`}
```

Comment:
`name` doesn't exist in this scope

```suggestion
              key={index}
```